frameworks/Python/web2py/setup.sh

```diff
@@ -0,0 +1,4 @@
+export PY2_ROOT=$IROOT/py2
+export PY2=$PY2_ROOT/bin/python
+
+sudo chmod a+w web2py/httpserver.pid & $PY2 web2py/web2py.py -a '' -i 127.0.0.1 -p 8080  &
```

Comment:
What's the requirement for sudo here? This script is run as testrunner, who should already have write permissions on the `web2py/` folder. If not, mark the folder as group and/or world writable. This sudo would break as we get closer to R11 and remove sudo permissions from testrunner